src/main/java/twilightforest/TFCommonProxy.java

```diff
@@ -36,6 +36,11 @@ public void doOnLoadRegistration() {
 	public World getClientWorld() {
 		return null;
 	}
+	
+	@SuppressWarnings("deprecation")
+	public String translate(String lang) {
+		return net.minecraft.util.text.translation.I18n.translateToLocal(lang);
```

Comment:
why is this in the proxy?

Reply:
This is here so that this *only* gets strictly called on the server without having to case it whenever there needs to be side-safe translation.